### PR TITLE
PRS in EMML1832 (no.1)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I have encoded several personal names in three distinct note on f. 1r and f. 3r in EMML1832.

In note 1 (a1) on folio 1r, I found a land grant to **PRS14233TawaldaM** in the time of ʿaqqābe saʿat **PRS14232TaklaIyasusMoa**. Personally, I was not able to identify the name of Tawalda Madḫen, because the writing on this folio is mostly effaced and some parts are hidden in the microfilm images because of a later paper note. However, I trust to Getatchew Haile, who was able to find this name and described it in his catalogue. The floruit of the abbot is in depth discussed in bm:Taddesse1970Abbots.

  ```<item xml:id="a1">
                           <locus target="#1r"/>
                           <desc type="LandGrant">Poorly legible record of a land grand by <persName ref="PRS3874eskender"/> to 
                              <persName ref="PRS14233TawaldaM"/> in <date notBefore="1488" notAfter="1499">6981 (= 1488/9 A.D.)</date>, 
                              when the ʿaqqābe saʿat was <persName ref="PRS14232TaklaIyasusMoa"/>. 
                              The last lines of the note are not visible behind a sheet of paper stuck to the page.</desc>
                           <q xml:lang="gez">በአኰቴተ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ወሀቦ፡ አነ፡ <persName ref="PRS3874eskender">እስክንድር፡</persName>
                              ወስመ፡ <supplied reason="lost">መን</supplied>ግሥትየ፡ 
                              <persName ref="PRS3874eskender"><supplied reason="lost">ቍ</supplied>ስጠንጢኖስ፡</persName> 
                              <gap reason="ellipsis" unit="lines" quantity="9"/></q>
                        </item>
```

Screenshot 2024_TawaldaMadhen

There are two different notes on folio 3r, both containing a number of names, including again PRS14232TaklaIyasusMoa and PRS14233TawaldaM, and also **PRS14234Marqoryos**, **PRS14235Matyas** and **PRS14236Kiros**, who are church officials at this time. 

There are two other names which I do not know whether they are place names or personal names. Getatchew Haile identified ገዜ as the name of an area given to Tawalda Madḫen. This could also be the case for an area called ብሕተ፡ አንበሳ፡ in the sense of "I, Takla ʾIyāsus Moʾa made ... Tawalda Madḫen to the guardian of the land Bǝḥta ʾAnbasā and of the land Gaze". Other solutions are possible, but I want to stick with Getatchew Haile and encode Gaze as a place name, and I am heading towards encoding ብሕተ፡ አንበሳ፡ as a place name as well. This name sounds strange as a place name and is not yet attested. It could be hypothesised as "the county of the one lion", as some monumental lion statues from pre-Christian times have been found in this region, such as the Lion of Kombolcha, that came to my mind.

```
<item xml:id="a4">
                           <locus target="#3r"/>
                           <desc type="Unclear">A note, the body of which has been erased, copied when 
                              <persName ref="PRS14232TaklaIyasusMoa">Takla ʾIyāsus Moʾa.</persName> was the ʿaqqābe saʿat.</desc>
                           <q xml:lang="gez"><gap reason="lost" unit="lines" quantity="5"/>
                              <persName ref="PRS14232TaklaIyasusMoa">ሞአ፡</persName> 
                              እንዘ፡ ቀሰ፡ ገበዝ፡ <persName ref="PRS14234Marqoryos">መርቀርዮስ፡</persName>
                              እንዘ፡ አፈ፡ ዓቃቤ፡ ሳዓት፡ <persName ref="PRS14235Matyas">ማትያስ፡</persName>
                              ወሊቀ፡ ዲያቆናት፡ ስብሐት፡ ለአብ፡ ወራቅ፡ ማስራ፡ <persName ref="PRS14236Kiros">ኪሮስ፡</persName>
                              ወበስምዓ፡ ኵሎሙ፡ ቅዱሳን፡ ነቢሮሙ፡ በጸፍጻፍ፡ ተጽሕፈ።</q>
                        </item>
```

```
<item xml:id="a5">
                           <locus target="#3r"/>
                           <desc type="LandGrant">A note in which the dues on two lands of <placeName ref="LOC2260DabraE"/>
                              are fixed by ʿaqqābe saʿat <persName ref="PRS14232TaklaIyasusMoa"/>.
                              <persName ref="PRS14233TawaldaM">Tawalda Madḫen</persName> according to this note, was the governor of that 
                              region <placeName ref="LOC7410Gaze"/>, which seems to be different from the 
                              <foreign xml:lang="gez">śǝyyuma mǝdr</foreign> that is also mentioned here.</desc>
                           <q xml:lang="gez">ሰራዕኩ፡ አነ፡ <persName ref="PRS14232TaklaIyasusMoa">ተክለ፡ ኢያሱስ፡ ሞአ፡</persName> ዓቃቤ፡ ሰዓት፡ 
                              ዘ<placeName ref="LOC3833Hayq">ሐይቅ፡</placeName>፳ወ፩፡ ሸማ፡ እምአለሸ፡ ወፈስስነ፡ ዘኢትስርዓ፡ ቀዳሚ፡ ለማኅቶት፡ ወለዕጣን፡ 
                              ወእንዘ፡ ዐቃቤ፡ ሰዓት፡ ለልየ፡ <persName ref="PRS14232TaklaIyasusMoa">ተክለ፡ ኢያሱስ፡ ሞአ፡</persName>
                              ወእንዘ፡ ቀሰ፡ ገበዝ፡ <persName ref="PRS14234Marqoryos">መርቀርዮስ፡</persName>
                              ወእንዘ፡ አፈ፡ ዐቃቤ፡ ሰዓት፡ <persName ref="PRS14235Matyas">ማትያስ፡</persName>
                              ወሊቀ፡ ዲያቆናት፡ ስብሐት፡ ለአብ፡ ወእንዘ፡ ረቍማስራ፡ <persName ref="PRS14236Kiros">ኪሮስ፡</persName>
                              ወስዮመ፡ ምድርሂ፡ <placeName ref="LOC00000000000000000">ብሕተ፡ አንበሳ፡</placeName>
                              ወምድረ፡ <placeName ref="LOC7410Gaze">ገዜ፡</placeName> 
                              <persName ref="PRS14233TawaldaM">ተወልደ፡ መድኅን፡ </persName>ዘሄደ፡ ወዘተአገለ፡ ወዘወለጠ፡ በአፈ፡ አብ፡ ወወልድ፡ 
                              ወመንፈስ፡ ቅዱስ፡ ውጉዘ፡ ይኩን፡</q>
                        </item>
```

Screenshot 2024_Behta Anbasa

### CODE REVIEW

File: new/PRS14232TaklaIyasusMoa.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተክለ፡ ኢያሱስ፡ ሞአ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Takla ʾIyāsus Moʾa</persName>
+                    <occupation type="ecclesiastic">ʿaqqābe saʿat</occupation>
+                    <floruit notBefore="1450" notAfter="1535">Succeeded <persName ref="PRS0000000000">Tasfā Giyorgis</persName>
+                        in <date when="1486">1486</date> as ʿaqqābe saʿat of <placeName ref="LOC2260DabraE"/>.</floruit>
```
Comment (eu-genia) [inline new/PRS14232TaklaIyasusMoa.xml:52]: LOC2260DabraE stands for Dabra Ǝgziʾabǝḥer, a minor church within the Hayq Estifanos complex

We used to have LOC2288DabraH for DHE which was deleted when INS0327DHE was created. (it was later decided that LOC should not be deleted, but for now we only have INS for Dabra Hayq Estifanos)

You need to refer to INS0327DHE throughout

File: new/PRS14235Matyas.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ማትያስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Mātyās</persName>
+                    <occupation type="ecclesiastic">ʾafa ʿaqqābe saʿat</occupation>
+                    <floruit notBefore="1450" notAfter="1535">Served as ʾafa ʿaqqābe saʿat in <placeName ref="LOC2260DabraE"/> in the time
```
Comment (eu-genia) [inline new/PRS14235Matyas.xml:51]: INS0327DHE

File: new/PRS14236Kiros.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ኪሮስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Kiros</persName>
+                    <occupation type="ecclesiastic">rāq māsrā</occupation>
+                    <floruit notBefore="1450" notAfter="1535">Served as rāq māsrā in <placeName ref="LOC2260DabraE"/> in the time of
```
Comment (eu-genia) [inline new/PRS14236Kiros.xml:51]: INS0327DHE

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14232TaklaIyasusMoa.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14232TaklaIyasusMoa.xml:52]: OK. Thanks for clarification!

**Review by eu-genia — APPROVED** [review]

**Pull request merged**